rollup.config.js

```diff
@@ -0,0 +1,124 @@
+import fs from "fs";
+import cjs from 'rollup-plugin-cjs-es';
+import {terser} from 'rollup-plugin-terser';
+import babel from '@rollup/plugin-babel';
+import resolve from "@rollup/plugin-node-resolve";
+import json from "@rollup/plugin-json";
+import iife from "rollup-plugin-iife";
+import alias from "@rollup/plugin-alias";
+import re from "rollup-plugin-re";
+import analyzer from "rollup-plugin-analyzer";
+import inject from "@rollup/plugin-inject";
+import esInfo from "rollup-plugin-es-info";
+import {visualizer} from "rollup-plugin-visualizer";
+
+const DEBUG = process.env.DEBUG === "1";
+
+export default {
+  input: {
+    "stylelint-bundle.min": 'index.js'
+  },
+  output: [
+    {
+      dir: "dist",
+      format: 'es',
+      sourcemap: true,
+      freeze: false
+    }
+  ],
+  // shimMissingExports: true,
+  plugins: [
+    re({
+      patterns: [
+        {
+          match: /.*/,
+          test: /const _?importLazy\s*=.+?;/g,
+          replace: ''
+        },
+        {
+          match: /.*/,
+          test: /\bimportlazy\(/gi,
+          replace: 'require('
+        },
+        {
+          match: /lib\/rules\/function-no-unknown\/index\.js/,
+          test: /fs\.readFileSync\(functionsListPath\.toString\(\), 'utf8'\)/,
+          replace: JSON.stringify(fs.readFileSync(require.resolve('css-functions-list/index.json'), 'utf8')),
+        },
+      ]
+    }),
+    alias({
+      entries: {
+        util: require.resolve("./shim/util"),
+        tty: require.resolve("./shim/tty"),
+        os: require.resolve("./shim/os"),
+        [require.resolve("stylelint/lib/getConfigForFile")]: require.resolve("./shim/getConfigForFile"),
+        [require.resolve("stylelint/lib/isPathIgnored")]: require.resolve("./shim/isPathIgnored"),
+        ...Object.fromEntries([
+          "cosmiconfig",
+          "css-functions-list",
```

Comment:
I thought this list helps some meaningful rule to warn about unknown css functions.

Reply:
I thought it was included via `rollup-plugin-re` but seems that it was not.